build.sbt

```diff
@@ -24,15 +70,81 @@ lazy val common = (project in file("common"))
-    libraryDependencies ++= Dependencies.commonDependencies
+    libraryDependencies ++= Seq(
+      "com.google.api-client" %  "google-api-client" % googleApiClientVersion,
+      "com.google.apis" % "google-api-services-youtube" % youTubeApiClientVersion,
+      "com.gu" %% "pan-domain-auth-play_2-5" % pandaVersion,
+      "com.gu" %% "pan-domain-auth-verification" % pandaVersion,
+      "com.gu" %% "pan-domain-auth-core" % pandaVersion,
+      "com.gu" %% "panda-hmac" % pandaHmacVersion,
+      PlayImport.ws,
+      "com.gu" %% "atom-publisher-lib" % atomMakerVersion,
+      "com.gu" %% "atom-publisher-lib" % atomMakerVersion % "test" classifier "tests",
+      "com.gu" %% "atom-manager-play" % atomMakerVersion,
+      "com.gu"  %% "atom-manager-play" % atomMakerVersion % "test" classifier "tests",
+      "com.google.guava" % "guava-jdk5" % guavaVersion,
+      "com.google.http-client" % "google-http-client-jackson2" % googleHttpJacksonVersion,
+      "com.google.oauth-client" % "google-oauth-client-jetty" % googleOauthVersion,
+      "com.google.code.findbugs" % "jsr305" % googleBugsVersion,
+      "com.google.http-client" % "google-http-client" % googleHttpVersion,
+      "commons-logging" % "commons-logging" % commonsLoggingVersion,
+      "org.apache.httpcomponents" % "httpclient" % apacheHttpClientVersion,
+      "org.apache.httpcomponents" % "httpcore" % apacheHttpCoreVersion,
+      "com.typesafe" % "config" % typesafeConfigVersion,
+      "com.amazonaws" % "aws-lambda-java-core" % awsLambdaCoreVersion,
+      "com.amazonaws" % "aws-java-sdk-s3" % awsVersion,
+      "com.amazonaws" % "aws-java-sdk-dynamodb" % awsVersion,
+      "org.cvogt" %% "play-json-extensions" % playJsonExtensionsVersion,
+      "net.logstash.logback" % "logstash-logback-encoder" % logstashLogbackEncoderVersion,
+      "com.gu" % "kinesis-logback-appender" % kinesisLogbackAppenderVersion,
+      "com.amazonaws" % "aws-java-sdk-elastictranscoder" % awsVersion,
+      "com.gu" %% "scanamo" % scanamoVersion,
+      "com.squareup.okhttp" % "okhttp" % okHttpVersion,
+      "org.scalatest" %% "scalatest" % scalaTestVersion % "test",
+      "org.scalacheck" %% "scalacheck" % scalaCheckVersion % "test", // to match ScalaTest version
+      "com.amazonaws" % "aws-java-sdk-sns" % awsVersion,
+      "com.amazonaws" % "aws-java-sdk-sqs" % awsVersion,
+      "com.gu" %% "editorial-permissions-client" % permissionsClientVersion,
+      "com.amazonaws" % "aws-java-sdk-stepfunctions" % awsVersion,
+      "com.amazonaws" % "aws-java-sdk-ses" % awsVersion,
+      "com.gu" %% "content-api-client-aws" % capiAwsVersion
+
+    )
   )
 
 lazy val app = (project in file("."))
   .dependsOn(common)
   .enablePlugins(PlayScala, SbtWeb, BuildInfoPlugin, JDebPackaging)
   .settings(commonSettings,
     name := "media-atom-maker",
-    libraryDependencies ++= Dependencies.appDependencies,
+    libraryDependencies ++= Seq(
+      "com.gu" %% "pan-domain-auth-play_2-5" % pandaVersion,
```

Comment:
Where we already have the dependency in `common` we don't need to re-state it here